<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordForm</class>
 <widget class="QDialog" name="PasswordForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="password_label">
     <property name="text">
      <string>Current Password</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="curpass">
     <property name="inputMask">
      <string notr="true"/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>New Password</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="newpass">
     <property name="inputMask">
      <string notr="true"/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Confirm Password</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="confpass">
     <property name="inputMask">
      <string notr="true"/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
